week-4/Homework/mandatory/4-groceries.js

```diff
@@ -6,7 +6,9 @@
 let groceriesToBuy = [];
 
 let groceryList = {
-  item1: "",
-  item2: "",
-  item3: ""
+  item1: "Potatoes",
+  item2: "Orange Juice",
+  item3: "Rice"
 };
+Object.values(groceryList).map(item => groceriesToBuy.push(item));
```

Comment:
i did exactly the same i think and i just saw Alisinias code and it is hard to give feedback because everything is similar in this exercises. i am glad you did it